docs/reference/aggregations/pipeline/normalize-aggregation.asciidoc

```diff
@@ -0,0 +1,121 @@
+[role="xpack"]
+[testenv="basic"]
+[[search-aggregations-pipeline-normalize-aggregation]]
+=== Normalize Aggregation
+
+A parent pipeline aggregation which calculates the specific normalized/rescaled value for a specific bucket value.
+
+==== Syntax
+
+A `normalize` aggregation looks like this in isolation:
+
+[source,js]
+--------------------------------------------------
+{
+    "normalize": {
+        "buckets_path": "normalized",
+        "normalizer": "percent_of_sum"
+    }
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+[[normalizer_pipeline-params]]
```

Comment:
Should we make a note somewhere that this pipeline always uses a `skip` gap policy?

Reply:
good call!